content/en/workshop/index.adoc

```diff
@@ -19,13 +19,13 @@ Association for Computational Linguistics] in Bangkok, Thailand.  Our primary
 aim is to provide a new venue to foster research on computational linguistics
 in Turkic languages.  By co-locating with ACL, we seek to facilitate
 interaction among researchers from diverse backgrounds, encouraging the
-exchange of ideas and the presentation of cutting-edge work in natural language
```

Comment:
I'd be more in favor of "the presentation of" since "exchange of ideas" is a noun as well.

Reply:
Either way reads more or less reasonable. Changing